packages/legacy/core/App/localization/fr/index.ts

```diff
@@ -117,27 +123,27 @@ const translation = {
         "Attestation": "J'ai lu, je comprends et j'accepte les conditions."
     },
     "Tour": {
-        "GuideTitle": "Welcome to Aries Bifold (FR)",
-        "WouldYouLike": "Would you like some guidance on how to use Aries Bifold? (FR)",
-        "UseAppGuides": "Use app guides (FR)",
-        "DoNotUseAppGuides": "Don't use app guides (FR)",
-        "AddAndShare": "Add and share credentials (FR)",
-        "AddAndShareDescription": "To add and use credentials you scan a QR code displayed by the service provider. (FR)",
-        "Notifications": "Notifications (FR)",
-        "NotificationsDescription": "After you scan a QR code, the credential offer or proof request will appear here, as well as other notable events. (FR)",
-        "YourCredentials": "Your credentials (FR)",
-        "YourCredentialsDescription": "Your added digital credentials appear here. You can review credential details and add and delete credentials. (FR)",
-        "Skip": "Skip (FR)",
-        "Next": "Next (FR)",
-        "Back": "Back (FR)",
-        "Done": "Done (FR)",
+        "GuideTitle": "Bienvenue à Aries Bifold",
+        "WouldYouLike": "Souhaitez-vous des conseils sur la façon d'utiliser Aries Bifold?",
+        "UseAppGuides": "Utiliser les guides d'applications",
+        "DoNotUseAppGuides": "N'utilisez pas les guides d'application",
```

Comment:
Assuming that this is the text on a button, can we shorten it to "N'utilisez pas les guides"